draft/2022-07-27-this-week-in-rust.md

```diff
@@ -153,23 +141,25 @@ Rusty Events between 2022-07-27 - 2022-08-24 🦀
 
 ### North America
 
-* 2022-07-26 | Austin, TX, US | [ATX Rustaceans](https://www.meetup.com/atx-rustaceans/)
-    * [**First Official Meetup - July 2022**](https://www.meetup.com/atx-rustaceans/events/285878081/)
 * 2022-07-27 | Austin, TX, US | [Rust ATX](https://www.meetup.com/rust-atx/)
     * [**Rust Lunch**](https://www.meetup.com/rust-atx/events/287009519/)
-* 2022-08-05 | Portland, OR, US | [RustConf](https://rustconf.com/)
+* 2022-08-05 | Portland, OR, US + Virtual | [RustConf](https://rustconf.com/)
     * [**RustConf 2022**](https://rustconf.com/)
```

Comment:
It sounds like RustConf registration gives priority consideration for the Unconf, but is not required for attendance. Suggesting the inclusion here. 
```suggestion
    * [**RustConf 2022**](https://rustconf.com/)
* 2022-08-06 | Portland, OR, US | [Rust Project Teams](https://www.rust-lang.org/governance)
    * [**RustConf 2022 PostConf Unconf**](https://www.eventbrite.com/e/rustconf-postconf-unconf-registration-373057423797) | [Blog post](https://blog.rust-lang.org/2022/06/28/rust-unconference.html)
```